DAOS-17679 test: Launch test to enable Bullseye

dinghwah opened the pull request:
Skip-fnbullseye: false
Skip-bullseye: false
Skip-python-bandit: true
Skip-build-EL9-rpm: true
Test-tag: test_pool_autotest
Allow-unstable-test: true
Doc-only: false

### Steps for the author:

* [ ] Commit message follows the [guidelines](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Appropriate [Features or Test-tag](https://daosio.atlassian.net/wiki/spaces/DC/pages/10984259629/Test+Tags) pragmas were used.
* [ ] Appropriate [Functional Test Stages](https://daosio.atlassian.net/wiki/spaces/DC/pages/12147556353/CI+Functional+Test+Stages) were run.
* [ ] At least two positive code reviews including at least one code owner from each category referenced in the PR.
* [ ] Testing is complete. If necessary, forced-landing label added and a reason added in a comment.

#### After all prior steps are complete:
* [ ] Gatekeeper requested (daos-gatekeeper added as a reviewer).

dinghwah committed:
Test to enable Bullseye

Skip-fnbullseye: false
Skip-bullseye: false
Skip-python-bandit: true
Skip-build-EL9-rpm: true
Test-tag: test_pool_autotest
Allow-unstable-test: true
Doc-only: false

Signed-off-by: Ding-Hwa Ho <ding-hwa.ho@hpe.com>

daosbuild3 commented:
Test stage NLT on EL 8.8 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net/job/daos-stack/job/daos/job/PR-17266/1/display/redirect

dinghwah committed:
Update Jenkinsfile with Bullseye

Skip-fnbullseye: false
Skip-bullseye: false
Skip-python-bandit: true
Test-tag: test_pool_autotest
Allow-unstable-test: true
Doc-only: false

Signed-off-by: Ding-Hwa Ho <ding-hwa.ho@hpe.com>

dinghwah committed:
Update Dockerfile.el.8 wih Bullseye

Skip-fnbullseye: false
Skip-bullseye: false
Skip-python-bandit: true
Test-tag: test_pool_autotest
Allow-unstable-test: true
Doc-only: false

Signed-off-by: Ding-Hwa Ho <ding-hwa.ho@hpe.com>

dinghwah committed:
Rerun with Bullseye key

Skip-fnbullseye: false
Skip-bullseye: false
Skip-python-bandit: true
Test-tag: test_pool_autotest
Allow-unstable-test: true
Doc-only: false

Signed-off-by: Ding-Hwa Ho <ding-hwa.ho@hpe.com>

dinghwah committed:
Retry

Skip-fnbullseye: false
Skip-bullseye: false
Skip-python-bandit: true
Test-tag: test_pool_autotest
Allow-unstable-test: true
Doc-only: false

Signed-off-by: Ding-Hwa Ho <ding-hwa.ho@hpe.com>
